chef/cookbooks/network/recipes/default.rb

```diff
@@ -581,6 +589,30 @@ def get_datapath_id_for_ovsbridge(bridge)
     end
   end
 
+  run_wicked_ifup = false
+
+  # This, when notified by the above "ifcfg" templates, sets run_wicked_ifup
+  # to true (which was initialized to false in the compile phase).
+  # run_wicked_ifup is later used as an "only_if" guard for the
+  # "wicked ifup all" call that is needs to happen when any of the config
+  # files got updated. The purpose of doing it this way (instead of notifying
+  # the "wicked-ifup-all" resource directly), is to make sure that the
+  # ifup is only run once after all ifcfg file have been update and
+  # independent of how many of them were changed.
+  ruby_block "wicked-ifup-required" do
+    block do
+      run_wicked_ifup = true
+    end
+    action :nothing
+  end
+
+  # Mark all changed interfaces as up, so wicked will keep them that way.
```

Comment:
Strictly speaking, I guess the comment is wrong: we simply ask wicked to mark all *configured* interfaces as up.